policy/centos9/scripts/upload-repo

```diff
@@ -6,10 +6,8 @@ pushd $(dirname $0)/..
 popd
 
 yum install -y epel-release
-yum install -y git python2-pip python-deltarpm
-pip install boto3==1.17.112
-pip install --cache-dir=/var/cache/pip/ \
-  git+git://github.com/Voronenko/rpm-s3.git@5695c6ad9a08548141d3713328e1bd3f533d137e
+yum install -y python3-pip
+pip install awscli==1.29.67
```

Comment:
Isn't this awscli v1? If yes, why do we need to use it instead of v2?

Reply:
@macedogm good point, I moved to awscliv2, see the latest [commit](https://github.com/rancher/rancher-selinux/pull/21/commits/93690b5944f4ad45f473ff7897a01fb0720ab50c).